composer.json

```diff
@@ -28,6 +28,7 @@
         "bamarni/composer-bin-plugin": "^1.8",
         "dama/doctrine-test-bundle": "^7.0|^8.0",
         "doctrine/common": "^3.2",
+        "doctrine/collections": "^1.7|^2.0",
```

Comment:
`doctrine/collections` < 1.7 is not compatible with phpunit 8.3, because of some return types